app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd = command.split()
+    if "mv" in cmd and len(cmd) == 3:
+        path = cmd[cmd.index("mv") + 1:]
```

Comment:
You can use split and unpacking
input_command, source_file_path, destination_file_path = cmd